src/cocaine-app/sched/recover_starter.py

```diff
@@ -0,0 +1,69 @@
+import logging
+import time
+
+import jobs
+from mastermind_core.config import config
+import storage
+
+
+
+logger = logging.getLogger('mm.sched.recover')
+
+
+class RecoveryStarter(object):
+
+    def __init__(self, job_processor, scheduler):
+        self.scheduler = scheduler
+        self.job_processor = job_processor
+        self.params = config.get('scheduler', {}).get('recover_dc', {})
+        scheduler.register_periodic_func(self._do_recover_dc, 60*15, starter_name="recover_dc")
```

Comment:
fixed in https://github.com/yandex/mastermind/pull/433/commits/934c87a29040c96b1422713157e19689a8ce9fcd